src/v0/destinations/zendesk/transform.js

```diff
@@ -294,13 +297,16 @@ async function getUserId(message, headers, baseEndpoint, type) {
 async function isUserAlreadyAssociated(userId, orgId, headers, baseEndpoint) {
   const url = `${baseEndpoint}/users/${userId}/organization_memberships.json`;
   const config = { headers };
-  const response = await axios.get(url, config);
-  if (
-    response.data &&
-    response.data.organization_memberships.length > 0 &&
-    response.data.organization_memberships[0].organization_id === orgId
-  ) {
-    return true;
+  try {
+    const response = await axios.get(url, config);
+    if (
+      response?.data?.organization_memberships?.[0]?.organization_id === orgId
+    ) {
+      return true;
+    }
+  } catch (error) {
+    logger.debug("Error :");
```

Comment:
[Small-change]
Let's capture this information in logs as error logs for `ZENDESK`. For now, let's add `ZENDESK` sub-string to the logs as well to serve as differentiator for us while searching for logs